docs/sponsorship/newsletter.rst

```diff
@@ -16,47 +17,38 @@ Audience
 Our audience is made up of technical writers and decision makers in the software documentation industry.
 The current newsletter stats are:
 
-* Over 5,000 subscribers
-* 37% open rate
-* 6% click rate
-* 70% of subscribers are based in the United States
-
-We don't have specific demographic data for our newsletter,
-but our conferences are made up of:
+* Over {{ newsletter_subs }} subscribers
+* 42% open rate
```

Comment:
Is it worth highlight just how high this number is compared to industry average? @goatlady

Reply:
The problem is that open rates are kind of dead as a metric since Apple started blocking tracking pixels by default and Google started caching images from bulk senders a couple of years ago. Your open rate is almost certainly much higher than that, but because most of the content is in the email itself rather than outbound links, MailChimp can't count the click-throughs as confirmed opens. 

Having said that, regardless of industry, an open rate of over 30% is considered excellent. There's an industry breakdown from 2022 data here: https://www.campaignmonitor.com/resources/knowledge-base/what-are-good-email-metrics/